SofaKernel/modules/SofaHelper/src/sofa/helper/ScopedAdvancedTimer.h

```diff
@@ -37,9 +39,10 @@ namespace sofa::helper
 ///     measurement recorded.
 struct SOFA_HELPER_API ScopedAdvancedTimer
 {
-    const char* message;
-    ScopedAdvancedTimer(const std::string& message);
-    ScopedAdvancedTimer( const char* message );
+    AdvancedTimer::IdStep m_id;
```

Comment:
I know this is not the topic of the PR and it is more about general c++ coding practice but including thousand of dependencies lines to get something that in fact is an integer is really problematic. I wonder what kind of design pattern should be used instead of these inner typedefs.